docs/contributing-to-airbyte/developing-locally.md

```diff
@@ -1,6 +1,18 @@
 # Developing Locally
 
-Airbyte uses `java 14` , `node 14` and `docker`
+The following technologies are required to build Airbyte locally.
+
+1. `Java 14`
+2. `Node 14`
+3. `Python 3.7`
+4. `Docker`
+5. `Postgresql`
+6. `Jq`
+7. `CMake`i
```

Comment:
Is that a typo with extra "i"?

I'm not sure I have "cmake" installed (but maybe "make" through XCode installation I guess...)

Reply:
oops. this often happens to me when I'm on the butterfly keyboard. good catch.

interestingly, I explicitly had to install `cmake` to do so.

Reply:
removing CMake results in this error for me:

```
  copying pyarrow/tests/data/parquet/v0.7.1.some-named-index.parquet -> build/lib.macosx-11.2-x86_64-3.7/pyarrow/tests/data/parquet
  creating build/lib.macosx-11.2-x86_64-3.7/pyarrow/tests/parquet
  copying pyarrow/tests/parquet/common.py -> build/lib.macosx-11.2-x86_64-3.7/pyarrow/tests/parquet
  copying pyarrow/tests/parquet/conftest.py -> build/lib.macosx-11.2-x86_64-3.7/pyarrow/tests/parquet
  copying pyarrow/tests/parquet/test_basic.py -> build/lib.macosx-11.2-x86_64-3.7/pyarrow/tests/parquet
  copying pyarrow/tests/parquet/test_data_types.py -> build/lib.macosx-11.2-x86_64-3.7/pyarrow/tests/parquet
  copying pyarrow/tests/parquet/test_dataset.py -> build/lib.macosx-11.2-x86_64-3.7/pyarrow/tests/parquet
  copying pyarrow/tests/parquet/test_datetime.py -> build/lib.macosx-11.2-x86_64-3.7/pyarrow/tests/parquet
  copying pyarrow/tests/parquet/test_metadata.py -> build/lib.macosx-11.2-x86_64-3.7/pyarrow/tests/parquet
  copying pyarrow/tests/parquet/test_pandas.py -> build/lib.macosx-11.2-x86_64-3.7/pyarrow/tests/parquet
  copying pyarrow/tests/parquet/test_parquet_file.py -> build/lib.macosx-11.2-x86_64-3.7/pyarrow/tests/parquet
  copying pyarrow/tests/parquet/test_parquet_writer.py -> build/lib.macosx-11.2-x86_64-3.7/pyarrow/tests/parquet
  running build_ext
  creating /private/var/folders/4h/k452x8ns06q7w_w7wkb3j16h0000gn/T/pip-install-cijvebo1/pyarrow/build/temp.macosx-11.2-x86_64-3.7
  -- Running cmake for pyarrow
  cmake -DPYTHON_EXECUTABLE=/Users/davinchia/.pyenv/versions/3.7.9/bin/python3.7 -DPython3_EXECUTABLE=/Users/davinchia/.pyenv/versions/3.7.9/bin/python3.7  -DPYARROW_BUILD_CUDA=off -DPYARROW_BUILD_FLIGHT=off -DPYARROW_BUILD_GANDIVA=off -DPYARROW_BUILD_DATASET=off -DPYARROW_BUILD_ORC=off -DPYARROW_BUILD_PARQUET=off -DPYARROW_BUILD_PLASMA=off -DPYARROW_BUILD_S3=off -DPYARROW_BUILD_HDFS=off -DPYARROW_USE_TENSORFLOW=off -DPYARROW_BUNDLE_ARROW_CPP=off -DPYARROW_BUNDLE_BOOST=off -DPYARROW_GENERATE_COVERAGE=off -DPYARROW_BOOST_USE_SHARED=on -DPYARROW_PARQUET_USE_SHARED=on -DCMAKE_BUILD_TYPE=release /private/var/folders/4h/k452x8ns06q7w_w7wkb3j16h0000gn/T/pip-install-cijvebo1/pyarrow
  error: command 'cmake' failed with exit status 1
```

Reply:
Ah yes, then it's related to pyarrow, since on my machine, I don't rebuild it completely but managed to `pip install pyarrow` binaries directly... but it's true I haven't tried using parquet yet so it might not work... I don't know 😆